src/ExceptionsTask.java

```diff
@@ -0,0 +1,53 @@
+import java.io.*;
+import java.util.Scanner;
+
+public class ExceptionsTask {
+	private String path="some path";
+	private void tryWithResources() {
+
+		try (BufferedReader br = new BufferedReader(new FileReader(path))) {
+				String line = br.readLine();
+		}catch (IOException e){
+			e.printStackTrace();
+		}
+	}
+	private void nestedTry(){
+		Scanner sc = null;
+		try
+		{
+			try{
+				sc = new Scanner(new File(path));
+			}catch (FileNotFoundException e){
+				e.printStackTrace();
+			}
+			try {
+				System.out.println(sc.hasNext());
+			}catch (NullPointerException e){
```

Comment:
Is this a suitable kind of Exception for code in try-block? Please, add the appropriate code or change the exception type.